resources/views/internlink/internship.blade.php

```diff
@@ -0,0 +1,86 @@
+@extends('layouts.panel')
+@section('panel-cols', '10')
+@section('panel-offset', '1')
+@section('panel-heading', 'Internship Profile')
+@section('panel-body')
+    <table id="profile">
+        <tr>
+            <td id="col1">Name</td>
+            <td id="col2">{!! $member_details['name'] !!}</td>
+        </tr>
+        @if(isset($member_details['mobile_uk']))
+            <tr>
+                <td id="col1">UK Mobile</td>
+                <td id="col2">{!! $member_details['mobile_uk'] !!}</td>
+            </tr>
+        @endif
+
+        @if(isset($member_details['mobile_home']))
+            <tr>
+                <td id="col1">UK Mobile</td>
+                <td id="col2">{!! $member_details['mobile_home'] !!}</td>
+            </tr>
+        @endif
+
+        @if(isset($member_details['email_hermes']))
+            <tr>
+                <td id="col1">UK Mobile</td>
+                <td id="col2">{!! $member_details['email_hermes'] !!}</td>
+            </tr>
+        @endif
+
+        @if(isset($member_details['email_other']))
+            <tr>
+                <td id="col1">UK Mobile</td>
+                <td id="col2">{!! $member_details['email_other'] !!}</td>
+            </tr>
+        @endif
+
+        <tr>
+            <td id="col1">Role</td>
+            <td id="col2">{!! $internship->role_name !!}</td>
+        </tr>
+
+        <tr>
+            <td id="col1">Company</td>
+            <td id="col2">{!! $internship->company_name !!}</td>
+        </tr>
+        <tr>
+            <td id="col1">Location</td>
+            <td id="col2">{!! $internship->location !!}</td>
+        </tr>
+        <tr>
+            <td id="col1">Start Date</td>
+            <td id="col2">{{ date('F Y', strtotime($internship->start_date)) }}</td>
```

Comment:
You could put this in the model. Try putting `start_date` and `end_date` in the `$dates` array of the model (https://laravel.com/docs/5.3/eloquent-mutators#date-mutators). This will parse it as a Carbon object. You might need to set a custom `$dateformat`. Hopefully it works even though you only have year/month. Then you should be able to do something like ` {{ $internship->start_date->format('F Y') }}` or something? There may be an even better solution here but at this stage it's balancing the tradeoff of the nicest, most elegant code [you only need to learn it once] v.s. actually getting it out the door and working...